src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -83,9 +95,67 @@
     }
 @end
 
+@private clientStreamingMethodSampleCode(apiMethod)
+    $requests = [$request];
+    
+    // Write data to server and wait for a response
+    $streamingResponse = {@methodCallSampleCode(apiMethod)};
+    $result = $streamingResponse->writeAllAndWait($requests);
```

Comment:
I like `$pullStream`, and perhaps also `$pushStream` and `$bidiStream`?

As for the GAX types, they could be named similarly.